### ISSUE TITLE
Join sequential paths

### ISSUE THREAD

**Issue by RazrFalcon** [RazrFalcon/svgcleaner#41]
```svg
<path d="m 50 50 l 10 10"/>
<path d="m 30 30 l 60 60"/>
```
into
```svg
<path d="m 50 50 l 10 10 m 30 30 l 60 60"/>
```

Main problem is in detecting that all attributes is equal.

**Comment by GreLI** [comment]
Not quite. Main problem is that if paths have intersections they may substract, so you have to check at least for intersections. In svgo it's done through checking convex hulls intersections by GJK algorythm.

**Comment by RazrFalcon** [comment]
Hello =)

Yes. This idea is mainly for non-fill paths. I didn't test it yet.
